### ISSUE TITLE
Create Web Scraper- for extracting tutorials 

### ISSUE THREAD

**Issue by Bhupesh-V** [Bhupesh-V/CoderBot#1]

**Comment by bicep** [comment]
Hello! I'm interested to give this a shot. Could you give some direction? :)

**Comment by Bhupesh-V** [comment]
@bicep 
Have you ever done web scraping 
Even if not ,try looking to some tutorials
The goal is to extract the course links from [this](https://hackr.io) website
The user specifies the language ,the bot extracts     the appropriate links and send to user.

This is the initial idea
If you have some other suggestions feel free to tell me 😋🤟

**Comment by RMPR** [comment]
@bicep still working on this ?

**Comment by bicep** [comment]
Yes I would love to, I just need to know how to generate a temporary token when I don't own the bot :(
But if you manage to get it working then I go ahead! let me know how you get it to work, I'm getting a polling error when I run `npm start` and I can't generate a new token.

**Comment by Bhupesh-V** [comment]
@bicep and @RMPR 
I m thinking to undeploy the bot currently
So that we can fix some bugs
And I will commit the Original API token.
What you say ??

**Comment by bicep** [comment]
Sounds good! Let me know when you do that!

**Comment by RMPR** [comment]
Good, I'm listening to you.

**Comment by Bhupesh-V** [comment]
@bicep and @RMPR
I have undeployed the bot
You can start working on it
It will now run locally
